modules/installation-guide/partials/proc_configuring-the-number-of-workspaces-a-user-can-run-using-the-operator.adoc

```diff
@@ -0,0 +1,40 @@
+
+[id="configuring-the-number-of-workspaces-a-user-can-run-using-the-operator_{context}"]
+= Configuring the number of workspaces a user can run using the Operator
+
+This procedure describes how to run more than one workspace simultaneously using the Operator. 
+
+.Prerequisites
+* An instance of `{prod-short}` installed using the Operator.
+* The combination of PVC strategy and access mode meets the following criteria:
++
```

Comment:
why do we need mentions of `PVC strategy`?

Reply:
See above ^